mileage_tracker/views.py

```diff
@@ -0,0 +1,16 @@
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from mileage_tracker.forms import MileageForm
+from mileage_tracker.models import DistanceToWork
+
+# Create your views here.
+@login_required(login_url='login/')
```

Comment:
You don't need to specify the login view here, it will redirect to the url configured inside setting.py